core/variable.h

```diff
@@ -21,12 +22,19 @@
 
 namespace scipp::core {
 
+template <class T>
+using sparse_container = boost::container::small_vector<T, 8>;
+template <class T> struct is_sparse_container : std::false_type {};
+template <class T>
+struct is_sparse_container<sparse_container<T>> : std::true_type {};
+
 class Variable;
 template <class... Known> class VariableConceptHandle_impl;
 // Any item type that is listed here explicitly can be used with the templated
 // `transform`, i.e., we can pass arbitrary functors/lambdas to process data.
 using VariableConceptHandle =
-    VariableConceptHandle_impl<double, float, int64_t, Eigen::Vector3d>;
+    VariableConceptHandle_impl<double, float, int64_t, Eigen::Vector3d,
+                               sparse_container<double>>;
```

Comment:
I guess at some point we will want to also have `int64_t` in the sparse container, for ex for times in nanoseconds.

Reply:
Yes, there will be a few more types in the handle eventually. For now I just added what was necessary to get the tests passing.